tests/integration/SlicePhaseTest.cpp

```diff
@@ -54,51 +56,53 @@ TEST_F(SlicePhaseTest, Cube)
     MeshGroup& mesh_group = scene.mesh_groups.back();
 
     const FMatrix4x3 transformation;
-    //Path to cube.stl is relative to CMAKE_CURRENT_SOURCE_DIR/tests.
-    ASSERT_TRUE(loadMeshIntoMeshGroup(&mesh_group, "integration/resources/cube.stl", transformation, scene.settings));
+    // Path to cube.stl is relative to CMAKE_CURRENT_SOURCE_DIR/tests.
+    ASSERT_TRUE(loadMeshIntoMeshGroup(
+      &mesh_group, std::filesystem::path(__FILE__).parent_path().append("resources/cube.stl").c_str(), transformation, scene.settings));
```

Comment:
Indent with 4 spaces.

Instead of breaking it up like this with an assert and `loadMeshIntoMeshGroup` function call vs the parameters of the call, I'd suggest breaking it up in the calculation of that path and then have the function calls be on one line.